### DISCUSSION TITLE
Not Understanding Batch Importing

### DISCUSSION THREAD

**Discussion by aeroeng14** [Gorgious56/asset_browser_utilities#99] [Q&A] [answered]
I'm trying to batch import meshes from multiple files to add them all as assets in 1 file, but the option doesn't exist under File->Import. Did it move or am I missing how to use the addon?

**Comment by Gorgious56** [comment]
Hello, strange as this sounds this feature is not part of the addon, but I'll add it to the todo list.

**Reply by aeroeng14** [reply]
Apologies. I was searching for a way to do what I described and found this post: https://blender.stackexchange.com/questions/245555/batch-add-to-asset-browser

And reading your answer seemed like the addon could do that. I must've misinterpreted what you were describing.

**Comment by Gorgious56** [comment, marked as answer]
Alright, no problem ! This BSE thread was actually one of the first iterations of my addon which has evolved a lot since then.

The feature to batch generate previews is still there, but the features have been moved from the File > Import menu to the asset browser editor, because as you experienced it, it was quite misleading since it doesn't actually import anything. You can batch generate previews for example with 
![image](https://user-images.githubusercontent.com/25156105/187631837-82eb84cb-5f80-4fc9-b994-988963e138b5.png)

But you can't yet batch import assets in the current file.

I hope that explanation clarifies your problem !

Cheers

**Reply by aeroeng14** [reply]
It does, thank you! I'll look at scripting all of my objects into one file, then running your addon. By the way I've used it for replacing a bunch of previews that Blender made goofy and it works great! Best of luck with future features and I'll be following your progress.
